src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -18,65 +18,75 @@ To better understand this page, we recommend you first read our [introduction to
 
-A consensus mechanism in a cryptoeconomic system also helps prevent certain kinds of economic attacks. In theory, an attacker can compromise consensus by controlling 51% of the network. Consensus mechanisms are designed to make this "51% attack" unfeasible. Different mechanisms are engineered to solve this security problem in different ways.
+A consensus mechanism in a cryptoeconomic system also helps prevent certain kinds of economic attacks. In Nakamoto consensus, an attacker can compromise the database (known as a "safety violation") by controlling 51% of the network hash power. Consensus mechanisms are designed to make this "51% attack" unfeasible. Different protocols are engineered to solve this security problem in different ways.
 
 <YouTube id="dylgwcPH4EA" />
 
 ## Types of consensus mechanisms {#types-of-consensus-mechanisms}
 
-### Proof-of-work {#proof-of-work}
+### Nakamoto Consensus {#nakamoto-consensus}
 
-Ethereum, like Bitcoin, currently uses a **proof-of-work (PoW)** consensus protocol.
+Ethereum, like Bitcoin, currently uses the Nakamoto, or "Longest Chain", consensus protocol.
 
 #### Block creation {#pow-block-creation}
 
-Proof-of-work is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/), who compete to create new blocks full of processed transactions. The winner shares the new block with the rest of the network and earns some freshly minted ETH. The race is won by whosever computer can solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle is the work in "proof-of-work".
+Nakamoto consensus is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/) who compete to create new blocks full of processed transactions. They do this through a process known as "proof-of-work" mining (PoW). In PoW, miners are creating random numbers, called a nonce, and plugging that into a hash function to see if they produce a hash which satisfies a condition difficult enough to show they've put work into solving the problem. Once discovered, the miner who found the nonce proposes a new block to the rest of the network. In this new block, the miner pays itself in ETH. If the network decides to continue to build on his block, it becomes part of the "canonical chain" and the miner has some freshly minted ETH.
+
+The race is won by whosever computer can solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle puts the work in "proof-of-work."
 
 #### Security {#pow-security}
 
 The network is kept secure by the fact that you'd need 51% of the network's computing power to defraud the chain. This would require such huge investments in equipment and energy; you're likely to spend more than you'd gain.
 
 More on [proof-of-work](/developers/docs/consensus-mechanisms/pow/)
 
-### Proof-of-stake {#proof-of-stake}
+### Casper FFG {#casper-ffg}
```

Comment:
is it right to jump straight into Casper-FFG here? FFG is the finality gadget that compares votes on pairs of checkpoints to give finality to the chain, but it's not the whole story of PoS consensus. There are also LMD-GHOST votes that vote on the head of the chain. Consensus over the head of the chain comes from LMD-GHOST, finality comes from FFG.

So the closest equivalent to ETH1/Bitcoin's Nakamoto consensus mechanism is LMD-GHOST, but instead of the longest chain (which is really the chain with the most accumulated work rather than the longest) it is the chain that has accumulated the greatest attestations backed by the most staked ether that becomes canonical. I think this nuance prevents LMD-GHOST from strictly being Nakamoto consensus. I guess it could be called "Nakamoto inspired"...!